sriov/tests/SR_IOV_RandomlyTerminate_DPDK/test_SR_IOV_RandomlyTerminate_DPDK.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+import random
+import time
+from sriov.common.utils import (
+    get_driver,
+    start_tmux,
+    create_vfs,
+    execute_and_assert,
+    execute_until_timeout,
+    bind_driver,
+    wait_tmux_testpmd_ready,
+    stop_testpmd_in_tmux,
+    get_pci_address,
+)
+
+
+def get_podman_cmd(name, cpus, dpdk_img, vf_pci):
+    tmux_cmd = (
+        f"podman run -it --name {name} --rm --privileged "
```

Comment:
We may want to add podman into required packages in README.md

Reply:
Yup, updated in latest commit. The caveat here is that this will likely become more generic going forward, so documentation will change further